cmake/uninstall.cmake

```diff
@@ -0,0 +1,25 @@
+MESSAGE(STATUS "Attempting to create uninstall target for make")
+SET(INSTALL_MANIFEST_PATH "${CMAKE_CURRENT_BINARY_DIR}/install_manifest.txt")
+IF(EXISTS ${INSTALL_MANIFEST_PATH})
+	MESSAGE(STATUS "install_manifest.txt found")
+	FILE(STRINGS ${INSTALL_MANIFEST_PATH} FILES_TO_REMOVE)
+	FOREACH(FILE_TO_REMOVE ${FILES_TO_REMOVE})
+		IF(EXISTS ${FILE_TO_REMOVE})
+			EXEC_PROGRAM(
```

Comment:
`exec_program` is deprecated, you should use [`execute_process`](https://cmake.org/cmake/help/latest/command/execute_process.html) instead.